modules/foreman_domain.py

```diff
@@ -87,36 +86,27 @@
 RETURN = ''' # '''
 
 try:
-    from ansible.module_utils.ansible_nailgun_cement import (
-        find_domain,
-        find_locations,
```

Comment:
This, orgs and proxies do make sense to me to keep since they often show up in APIs. Perhaps there needs to a compatible apypie helper?

Reply:
I thought about that (even had them at first), but then I thought that `find_foo(…)` is not that shorter than `find_resource_by_name('foo', …)` and having that for only a few resources is also weird?

Reply:
There is still the `thin=True` as well. Perhaps the normalization function should be factored out and do it there.